en/identity-server/next/docs/guides/users/user-stores/primary-user-store/configure-a-read-write-active-directory-user-store.md

```diff
@@ -30,7 +30,51 @@ Furthermore, please make sure to follow the steps mentioned in [Configure Active
 since SCIM is enabled by default from the WSO2 Identity Server 5.10.0 onwards.
 
 
+### Configuring a fresh server
+If you are configuring a server that has not been started yet, you need to update the claim mappings in
+`<carbon_home>/repository/conf/claim-config.xml`.
+
+!!! note
+    Following are some of the mandatory claims that you need to map with the user store attributes.
+
+      - `http://wso2.org/claims/username`
+      - `http://wso2.org/claims/userid`
+      - `http://wso2.org/claims/created`
+      - `http://wso2.org/claims/modified`
+
+### Configuring an already started server
+Before you change `deployment.toml` with above configurations, you need to change the claim mappings for the
+`PRIMARY` user store by navigating to the `User Attributes & Stores > Attributes` section of the console. After
+updating the mappings, shutdown the server, update the `deployment.toml` and restart the server to apply the
+configurations.
+
 !!! note
-    It is required to edit the claim mappings in WSO2 IS according to the user claims of the Active Directory version you have configured.<br />
-    Before starting the server, edit the `<IS_HOME>/repository/conf/claim-config.xml` configuration file and change the `AttributeID` of the `Created Time` and `Last Modified Time` claims to `whenCreated` and `whenChanged` respectively.
```

Comment:
Don't we need to mention the specific attributes that are needed to be mapped with below attributes?

Reply:
Updated